lastminutewf2020.yml

```diff
@@ -75,3 +75,41 @@
         msg: "Finished {{ version }}"
         priority: "notice"
       changed_when: false
+    - name: workaround nautilus auto-mounting
+      ansible.builtin.file:
+        path: /media
+        owner: root
+        group: root
+        mode: "0700"
+    - name: gvfs permission tweaking
+      ansible.builtin.file:
+        path: /usr/lib/gvfs/gvfs-{{ item }}
+        owner: root
+        group: root
+        mode: "0744"
+      loop:
+        - "udisks2-volume-monitor"
```

Comment:
Probably an item is missing? Otherwise the loop is not needed,

Reply:
Alternative is to add the gvfs in front and merge the loop here and below.

Reply:
hmm,  it was done this way,  because before there were move gvfs items.

Reply:
moved to a single loop